irs-api/src/main/resources/application.yml

```
@@ -129,6 +129,8 @@ resilience4j:
         baseConfig: default
 
 irs-edc-client:
+  callback:
+    mapping: ${EDC_TRANSFER_CALLBACK_MAPPING:/internal/endpoint-data-reference} # The callback endpoint mapping
```

Hi @ds-jhartmann, I think we talked about this in my last story that we don't need the env variables here because Spring does this. So IMHO it would be sufficient to have the following here, wouldn't it?
```suggestion
    mapping: /internal/endpoint-data-reference  # The callback endpoint mapping
```